readme.md

```diff
@@ -15,8 +15,12 @@ A command line script to print wirelessly in the Oden Intitute at UT Austin from
 
 - `oden_print -h` : help message
 - `oden_print -l` : list available printers on network
+- `oden_print -d` : list your default printer
+- `oden_print -m` : list your ssh machine 
+- `oden_print -# N printer_name file_to_print` : prints N copies of `file_to_print` on printer `printer_name`
 - `oden_print printer_name file_to_print` : prints `file_to_print` on printer `printer_name`
 
+Note that calling without specifying a printer will print on `<printer name>` 
```

Comment:
```suggestion
Note that calling without specifying a printer will print on `printer name`, which must be set in the function body by the user.
```
Remove angle brackets, and make it clear users will have to specify default printer.